paparazzi/src/main/java/app/cash/paparazzi/DeviceConfig.kt

```diff
@@ -261,7 +261,7 @@ data class DeviceConfig(
       xdpi = 440,
       ydpi = 440,
       orientation = ScreenOrientation.PORTRAIT,
-      density = Density.DPI_420,
+      density = Density.create(420),
```

Comment:
Density was changed from an enum to a class.

Reply:
Hmm, someone was asking for something like this.